pkg/api/index.go

```diff
@@ -120,11 +120,15 @@ func (hs *HTTPServer) getAppLinks(c *models.ReqContext) ([]*dtos.NavLink, error)
 			}
 
 			if include.Type == "dashboard" && include.AddToNav {
-				link := &dtos.NavLink{
-					Url:  hs.Cfg.AppSubURL + include.GetSlugOrUIDLink(),
-					Text: include.Name,
+				dboardPath := include.DashboardURLPath()
+
+				if dboardPath != "" {
```

Comment:
Maybe a warning logged here as well if it's empty?

Reply:
I think this will be too spammy, as this runs on every index refresh

Reply:
Good point :+1: